src/Tools/dotnet-dump/Program.cs

```diff
@@ -101,7 +101,8 @@ private static Argument DumpPath() =>
             new Argument<FileInfo>(
                 name: "dump_path")
             {
-                Description = "Name of the dump file to analyze."
+                Description = "Name of the dump file to analyze.",
```

Comment:
This is weird from a UX perspective - analyze with no target just feels weird to me. Analyze should ALWAYS have a target.

Reply:
To me the "analyze" verb means bring up the REPL (as opposed to "collect") so this makes sense to me as it allows other target types to be analyzed.

Reply:
My initial feeling is similar to Juan's. 

> as it allows other target types to be analyzed.

What other target types would we want to analyze? I may misunderstand but if those other targets aren't dumps then it feels out-of-place for the tool named dotnet-dump. I'm not opposed to supporting a debugging scenario that works with something else (like a live process?), I just wouldn't expect that functionality to be present in a tool named dotnet-dump.

Reply:
I'm backing out this dotnet-dump analyze changes.